src/comp/comp.py

```diff
@@ -24,49 +24,50 @@ def __repr__(self):
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with 'D':
 print("Starts with D:")
-a = [x.name for x in humans if x.name.startswith("D")]
+a = [human.name for human in humans if human.name.startswith("D")]
 print(a)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name ends in "e".
 print("Ends with e:")
-b = [x.name for x in humans if x.name.endswith("e")]
+b = [human.name for human in humans if human.name.endswith("e")]
 print(b)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with any letter between 'C' and 'G' inclusive.
 print("Starts between C and G, inclusive:")
 prefixes = ('C', 'D', 'E', 'F', 'G') 
-c = [x.name for x in humans if x.name.startswith(prefixes)]
+c = [human.name for human in humans if human.name.startswith(prefixes)]
 print(c)
 
```

Comment:
similar to what was done with the ages you can do this with the letter range rather than writing out each letter:

"human.name for human in humans if human.name[0] >= 'C' and human.name[0] <= 'G'"